test/spec/exceptions/application_validator.spec.js

```diff
@@ -318,13 +318,6 @@ describe('ApplicationValidator', () => {
           });
         });
       });
-      context('with unknown options', () => {
-        it('should fail', () => {
-          expect(() =>
-            validator.validate(new JDLApplication({ config: { ...basicValidApplicationConfig, toto: 42 } }))
-          ).to.throw(/^Unrecognised application option name: toto\.$/);
-        });
-      });
```

Comment:
I would have kept it but replaced the failure part with "it should send a debug message" (or something like that)

Reply:
I have replaced it with a different "should not throw an error"; although your message "it should send a debug message" seems maybe more appropriate but I couldn't think of a way to check this debug log message is actually printed in the test suite. Is there a way? :smile: